docs/source/authoring_flows.rst

```diff
@@ -535,14 +535,10 @@ View the `Move flow definition`_ in the Globus web app.
     :caption: Example Input
 
     {
-        "source": {
-          "id": "ddb59aef-6d04-11e5-ba46-22000b92c6ec",
-          "path": "/~/source-directory"
-        },
-        "destination": {
-          "id": "ddb59af0-6d04-11e5-ba46-22000b92c6ec",
-          "path": "/~/destination-directory"
-        }
+        "source_endpoint_id": "ddb59aef-6d04-11e5-ba46-22000b92c6ec",
+        "source_path": "/~/source-directory",
+        "destination_endpoint_id": "ddb59af0-6d04-11e5-ba46-22000b92c6ec",
+        "destination_path": "/~/destination-directory",
```

Comment:
We do want to advertise the collection format, so I'm going to close this PR but work to fix the issues that you called out.